src/liballoc/boxed.rs

```diff
@@ -479,6 +482,15 @@ impl From<Box<str>> for Box<[u8]> {
     }
 }
 
+#[allow(incoherent_fundamental_impls)]
+#[stable(feature = "box_into_non_null", since = "1.32.0")]
```

Comment:
This needs to be since 1.33 because the impl will land in nightly first, which is currently 1.33.0-nightly.

Reply:
Actually this needs to be 1.34 since master=>beta promotion will happen on the 15th of January and FCP is 10 days... 9 + 10 = 19,  19 > 15.